src/components/auth/ResetPasswordForm.jsx

```diff
@@ -0,0 +1,86 @@
+import React, { useState, useEffect } from 'react';
+import axios from 'axios';
+import FormInput from "../common/FormInput";
+import { useNavigate } from "react-router-dom";
+
+const ResetPasswordForm = () => {
+  const [password, setPassword] = useState('');
+  const [confirmPassword, setConfirmPassword] = useState('');
+  const [message, setMessage] = useState('');
+  const navigate = useNavigate();
+  const [token, setToken] = useState('');
+
+  useEffect(() => {
+    const urlToken = new URLSearchParams(window.location.search).get('token');
+    console.log('Extracted token from URL:', urlToken);
+    if (urlToken) {
+      setToken(urlToken);
+    } else {
+      console.error('No token found in URL');
+    }
+  }, []);
+
+  const handleSubmit = async (e) => {
+    e.preventDefault();
+    if (password !== confirmPassword) {
+      setMessage('Passwords do not match');
+      return;
+    }
+  
+    try {
+      const response = await axios.patch('http://localhost:8000/api/v1/reset-password', {
```

Comment:
There are a lot of instances of this base url through the front end code base. Could benefit from being moved to an environment variable. Let's discuss during standup.

Reply:
Yes, thank you. Valentina showed me frontend already have separate file with Data base request.